common4j/src/main/com/microsoft/identity/common/java/crypto/IStorageEncryptionManager.java

```diff
@@ -0,0 +1,13 @@
+package com.microsoft.identity.common.java.crypto;
+
+import com.microsoft.identity.common.java.exception.ClientException;
+
+import lombok.NonNull;
+
+/**
+ * Add helper functions which takes in parameter or produce results in a ready-to-store (String) form.
+ */
+public interface IStorageEncryptionManager extends IKeyAccessor {
```

Comment:
I believe we're going to change this to adapter model anyway based on the meeting

Reply:
yup. Changed to KeyAccessorStringAdapter.